advanced.md

```diff
@@ -45,7 +45,7 @@ Whenever you construct the `UwpFirmata` object yourself, you **must** call `star
 
 ```c#
 //member variables
-IStream bluetooth;
+IStream connection;
```

Comment:
Yes, the name of the variable conflicts with the other names. This is not critical, but consider this part of the documentation